pennylane/qchem/observable_hf.py

```diff
@@ -43,34 +42,34 @@ def fermionic_observable(constant, one=None, two=None, cutoff=1.0e-12):
     >>> ops
     [[], [0, 0], [0, 2], [1, 1], [1, 3], [2, 0], [2, 2], [3, 1], [3, 3]]
     """
-    coeffs = anp.array([])
+    coeffs = np.array([])
 
-    if constant != anp.array([0.0]):
-        coeffs = anp.concatenate((coeffs, constant))
+    if constant != np.array([0.0]):
```

Comment:
Probably better to qml.math.allclose here.

Reply:
Done.